docs/getting-started/first-traffic-split.md

```diff
@@ -6,14 +6,14 @@ The last super power :rocket: of Knative Serving we'll go over in this tutorial
 
 
 ## Creating a new Revision
-You may have noticed that when you created your Knative Service you assigned it a `revision-name`, "world". When your Service was created, Knative returned both a URL and a 'latest revision' name for your Knative Service. **But what happens if you make a change to your Service?**
+You may have noticed that when you created your Knative Service you assigned it a `revision-name`, "world". If you used `kn`, when your Service was created Knative returned both a URL and a "latest revision" for your Knative Service. **But what happens if you make a change to your Service?**
 
-??? question "What exactly is a `Revision`?""
-    You can think of a <a href="../../serving/#serving-resources" target ="blank_">`Revision` </a> as a stateless, autoscaling snapshot-in-time of application code and configuration.
+??? question "What exactly is a Revision?""
+    You can think of a <a href="../../serving/#serving-resources" target ="blank_">Revision </a> as a stateless, autoscaling, snapshot-in-time of application code and configuration.
 
-    A new `Revision` will get created each and every time you make changes to your Knative Service, whether you assign it a name or not. When splitting traffic, Knative splits traffic between different `Revisions` of your Knative Service.
+    A new Revision will get created each and every time you make changes to your Knative Service, whether you assign it a name or not. When splitting traffic, Knative splits traffic between different Revisions of your Knative Service.
```

Comment:
A new Revision is created every time you make changes to your Knative Service, whether you assign it a name or not. When splitting traffic, Knative splits traffic between different Revisions of your Knative Service.